packages/belos/src/BelosMinresIter.hpp

```diff
@@ -592,8 +594,8 @@ class MinresIter : virtual public MinresIteration<ScalarType,MV,OP> {
 
       // Update x:
       // x = x + phi*w;
-      MVT::MvAddMv( one, *cur_soln_vec, phi, *W_, *cur_soln_vec );
-      lp_->updateSolution();
+      //MVT::MvAddMv( one, *cur_soln_vec, phi, *W_, *cur_soln_vec );
```

Comment:
remove commented line, add comment that `updateSolution` does this?

Reply:
Sure, I can add that when I fix the solver for right-preconditioning.